tests/utilities/test_xla_device_utils.py

```diff
@@ -11,13 +11,17 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+import math
+import time
+
 import pytest
 
-from pytorch_lightning.utilities.xla_device_utils import XLADeviceUtils as xdu
+import pytorch_lightning.utilities.xla_device_utils as utils
```

Comment:
```suggestion
import pytorch_lightning.utilities.xla_device_utils as xla_utils
```
to be clear what utils...